src/Impl/RenderTarget/SfmlRenderTarget/RenderTarget.cpp

```diff
@@ -0,0 +1,94 @@
+#include "Impl/Graphics/RenderTarget/SfmlRenderTarget/RenderTarget.h"
+
+static sf::Color    getSFMLColor(const plug::Color& plug_color);
+static sf::Vector2f getSFMLVector2f(const plug::Vec2d& plug_vector);
+
+static void copyToSFMLVertex(sf::Vertex&         sf_vertex,
+                             const plug::Vertex& plug_vertex);
+static void copyToSFMLVertexArray(sf::VertexArray&         sf_array,
+                                  const plug::VertexArray& plug_array);
+static void copyToSFMLTexture(sf::Texture&         sf_texture,
+                              const plug::Texture& plug_texture);
+
+//=======================================================================
+
+void SfmlRenderTarget::draw(const plug::VertexArray& array)
+{
+  sf::VertexArray sf_array;
+  copyToSFMLVertexArray(sf_array, array);
+
+  m_target.draw(sf_array);
+}
+
+void SfmlRenderTarget::draw(const plug::VertexArray& array,
+                            const plug::Texture&     texture)
+{
+  sf::VertexArray sf_array;
+  copyToSFMLVertexArray(sf_array, array);
+
+  sf::Texture sf_texture;
+  copyToSFMLTexture(sf_texture, texture);
+
+  m_target.draw(sf_array, &sf_texture);
+}
+
+void SfmlRenderTarget::clear(plug::Color color)
+{
+  m_target.clear(getSFMLColor(color));
+}
+
+unsigned SfmlRenderTarget::getNativeHandle(void) const { return 0; }
```

Comment:
This is not how it is supposed to work...

Plus, there is an issue with this format. I recommend you check it out.
https://github.com/B05-232/plug-standard/issues/30